app/models/abilities/organization_member_ability.rb

```diff
@@ -44,27 +44,24 @@ def expense_sheet_abilities(organization_id)
     end
 
     def service_abilities(organization_id)
-      can(:read, Service, {
-            service_specification: {
-              organization_id: organization_id
-            }
-          })
+      base_scope = { service_specification: { organization_id: organization_id } }
 
-      can(:manage, Service, {
-            organization_agreed: true,
-            civil_servant_agreed: nil,
-            service_specification: {
-              organization_id: organization_id
-            }
-          })
+      can(:read, Service, base_scope)
+      can(:crud, Service, base_scope.merge(
```

Comment:
what is the difference between crud and manage?

Reply:
manage = everything, automatically, also custom actions
crud = equivalent to alias I defined which is only create read update and destroy

Reply:
Problem was because he could manage, he could confirm services which cannot be confirmed anymore

Reply:
aah smart